src/main/kotlin/xyz/wagyourtail/unimined/UniminedExtensionImpl.kt

```diff
@@ -9,22 +9,25 @@ import xyz.wagyourtail.unimined.api.source.task.MigrateMappingsTask
 import xyz.wagyourtail.unimined.api.minecraft.MinecraftConfig
 import xyz.wagyourtail.unimined.internal.mapping.task.MigrateMappingsTaskImpl
 import xyz.wagyourtail.unimined.internal.minecraft.MinecraftProvider
+import xyz.wagyourtail.unimined.internal.minecraft.patch.reindev.ReIndevProvider
 import xyz.wagyourtail.unimined.util.defaultedMapOf
 import xyz.wagyourtail.unimined.util.withSourceSet
 import java.net.URI
 import java.nio.file.Path
 
 open class UniminedExtensionImpl(project: Project) : UniminedExtension(project) {
 
-    override val minecrafts = defaultedMapOf<SourceSet, MinecraftConfig> {
-        MinecraftProvider(project, it)
-    }
+    override val minecrafts = mutableMapOf<SourceSet, MinecraftConfig>()
```

Comment:
This may be a point of contention, but it was necessary to make this work.

Let me know what you think!